lib/services/sync/sync_service.ts

```diff
@@ -794,6 +794,9 @@ export class SNSyncService extends PureService {
     if (response.status === INVALID_SESSION_RESPONSE_STATUS) {
       this.notifyEvent(SyncEvent.InvalidSession);
     }
+    if (response.status === EXPIRED_ACCESS_TOKEN_RESPONSE_STATUS) {
+      this.notifyEvent(SyncEvent.ExpiredAccessToken);
```

Comment:
It seems like renewing session token should be handled by either sync_service or session_manager directly.

Reply:
You're right. I'm checking for the status code on `sync_service`, but it would be more appropriate to do so from `session_manager` because this event could be triggered from other requests other than the sync ones (example `changePassword`).